pkg/operation/garden/garden.go

```diff
@@ -41,6 +42,22 @@ func New(projectLister gardenlisters.ProjectLister, namespace string) (*Garden,
 	}, nil
 }
 
+//Check annotation "dns.garden.sapcloud.io/hostedZoneID", return true if the field is satisfied
+func checkDNSHostedZoneID(annotations map[string]string) bool {
+	if annotations == nil {
+		return false
+	}
+
+	r := true
+	//DNSHostedZoneID is not needed for AlicloudDNS
```

Comment:
why? please add more comments here.

Reply:
Please see preious comment. Hosted Zone is not supported by Alicloud DNS.